doc/source/notebooks/advanced/convolutional.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
Honestly, notebook looks like a bunch of commands to run. No description of what's happening and anything what guides the user for deeper understanding why this kernel exists.